data/human/hails.txt

```diff
@@ -1096,6 +1096,50 @@ phrase "friendly civilian"
 		"deserves to be put down!"
 		"has to be replaced!"
 
+phrase "friendly civilian"
+	word
+		"Some"
+		"Some of my"
+	word
+		" "
+	word
+		"passengers"
+		"guests"
+		"VIP passengers"
+	word
+		" "
+	word
+		"ate asparagus"
+		"ate garlic"
+		"ate beans"
```

Comment:
Where/when did they eat the food? Onboard?

Reply:
I'll add some possible locations.